pkg/odo/cli/init/init.go

```diff
@@ -140,20 +140,12 @@ func (o *InitOptions) Run() (err error) {
 	if err != nil {
 		return err
 	}
-	scontext.SetComponentType(o.ctx, component.GetComponentTypeFromDevfileMetadata(devfileObj.Data.GetMetadata()))
 
 	starterInfo, err := o.clientset.InitClient.SelectStarterProject(devfileObj, o.flags, o.clientset.FS, o.contextDir)
 	if err != nil {
 		return err
 	}
 
-	// Set the name in the devfile but do not write it yet to disk,
-	// because the starter project downloaded at the end might come bundled with a specific Devfile.
-	name, err := o.clientset.InitClient.PersonalizeName(devfileObj, o.flags)
-	if err != nil {
-		return fmt.Errorf("Failed to update the devfile's name: %w", err)
-	}
-
```

Comment:
Please take care not to move this code

Reply:
What do you mean, not remove it or move it?

Reply:
I don't know exactly, but the only changes should be about telemetry. Here this code seems to have been moved after another non-telemetry block